packages/replayio/src/commands/record.ts

```diff
@@ -52,7 +55,12 @@ async function record(url: string = "about:blank") {
     logPromise(promise, {
       messages: {
         pending: "Uploading crash data...",
-        success: "Crash data uploaded successfully",
+        success: () => {
+          if (uploadableCrashes.some(recording => recording.uploadStatus === "failed")) {
+            return "Some of the crash data couldn't be uploaded";
+          }
+          return "Crash data uploaded successfully";
+        },
```

Comment:
This isn't exactly beautiful and I want to rewrite this somehow. 

Atm I'm a little bit brain-farted and can't figure out how to juggle this code better - I might consider making the `uploadRecodings` a deferred action to get synchronous access to the "return value". However, even that won't fix everything - the problem is that `uploadRecordings` "settles" everything so the final result is always a success (and that comes with a green tick in the generated message). I want to avoid adding some clunky rendering options.

Reply:
> Atm I'm a little bit brain-farted and can't figure out how to juggle this code better 

Lemme know if I can help any!